services/movie_session.py

```diff
@@ -42,3 +42,11 @@ def update_movie_session(
 
```

Comment:
You are using  ` QuerySet`  in `def get_movies_sessions(session_date: str = None) -> QuerySet:` but i can’t find import